CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+- Added support for Terraform ~0.12 [10]
```

Comment:
Lets use same notation as in other repos: 

> Supported version of Terraform is 0.12.

Reply:
Ok

Reply:
Fixed